hoomd/md/pair.py

```diff
@@ -1864,38 +1989,42 @@ class ReactionField(_Pair):
     If *use_charge* is set to True, the following formula is evaluated instead:
     .. math::
```

Comment:
```suggestion

    .. math::
```
Add an extra empty line here so the documentation renders correctly.